components/AppLayout/Widgets/Avatar.js

```diff
@@ -124,15 +121,12 @@ const OpenPeepsAvatar = withStyles(theme => ({
   // eslint-disable-next-line no-unused-vars
 }))(({ className, classes, avatarData, size, mdSize, ...rest }) => {
   return (
-    <div className={className} {...rest}>
-      <div className={classes.showcaseWrapper}>
-        <Peep
-          {...omit(avatarData, ['backgroundColor'])}
-          style={peepsStyles.peepStyle}
-          circleStyle={peepsStyles.circleStyle}
-          strokeColor="#000"
-        />
-      </div>
+    <div className={`${className} ${classes.showcaseWrapper}`} {...rest}>
```

Comment:
Use `cx` so that we won't end up with `undefined` in the `className` (which can be seen in snapshot)

```suggestion
    <div className={cx(className, classes.showcaseWrapper)} {...rest}>
```